.goreleaser.yml

```diff
@@ -0,0 +1,65 @@
+
+
+# https://goreleaser.com for documentation
+
+version: 2
+before:
+  hooks:
+    - go mod tidy
+git:
+  ignore_tags:
+    - "*rc*"
+builds:
+  - binary: '{{ .ProjectName }}_v{{ .Version }}'
+    env:
+      # goreleaser does not work with CGO, it could also complicate
+      # usage by users in CI/CD systems like HCP Terraform where
+      # they are unable to install libraries.
+      - CGO_ENABLED=0
+    mod_timestamp: '{{ .CommitTimestamp }}'
+    flags:
+      - -trimpath
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+    goos:
+      - freebsd
+      - windows
+      - linux
+      - darwin
+    goarch:
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: windows
+        goarch: arm
+archives:
+  - formats: [ 'zip' ]
+    name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+checksum:
+  extra_files:
+    - glob: 'terraform-registry-manifest.json'
+      name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
```

Comment:
We can  use a YAML anchor to define this block once and reuse it. This reduces duplication and makes future changes easier.

```
extra_files: &extra_files
  - glob: 'terraform-registry-manifest.json'
    name_template: '{{ .ProjectName }}_{{ .Version }}_manifest.json'
```

Then use it 
```
checksum:
  extra_files:
    <<: *extra_files
```